tests/model_serving/model_server/llmd/test_llmd_singlenode_precise_prefix_cache.py

```
@@ -67,6 +67,8 @@ def test_singlenode_precise_prefix_cache(
             delay_after_first_request=15,
         )
 
+        import pdb; pdb.set_trace()
```

Flake8 (7.3.0)

[error] multiple statements on one line (semicolon)

(E702)